SU-SWS/stanford_profile_helper
# D8CORE-4341: moving code from the person module.

# READY FOR REVIEW

# Summary
- Changing the new people grid when on a list.

# Needed By (Date)
- with the https://github.com/SU-SWS/stanford_person/pull/143 

# Urgency
- Med

# Steps to Test

1. Pull in the Person changes - https://github.com/SU-SWS/stanford_person/pull/143
2. have 3+ people
3. Make a list and verify that when it is in 2 or 3 cols it is no longer in a grid. When the list is in one cols, it need the grid.

# Affected Projects or Products
- People

# Associated Issues and/or People
- D8CORE-4341
- https://github.com/SU-SWS/stanford_person/pull/143

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
